src/js/unity-menu-script.js

```diff
@@ -5,12 +5,14 @@
       $('#main-menu').find('.expanded .menu-link').once('has-submenu').each(function () {
 
         var $submenu = $(this).next('.menu-main_sub');
+        var $sidemenu = $('#block-sidemenu').find('.menu');
 
         if ($submenu.length == 1) {
           var $content = $(this).text();
           var $link = $(this).attr("href");
 
-          $submenu.prepend('<li class="menu-item leaf title"><a href="'+$link+'" class="menu-link">'+$content+'</a></li>')
+          $submenu.prepend('<li class="menu-item leaf title"><a href="'+$link+'" class="menu-link">'+$content+'</a></li>');
```

Comment:
Not sure if picked up by the linter but separating string portions with a space when concatenating generally makes for a more readable format.

eg: `<a href="' + $link + '" class etc etc` instead of `<a href="'+$link+'" class`